src/main/java/co/org/osso/api/BookController.java

```diff
@@ -0,0 +1,38 @@
+package co.org.osso.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class BookController {
+
+    private final CustomerService customerService;
+    private final BookService bookService;
+
+    BookController(CustomerService customerService, BookService bookService){
+        this.customerService = customerService;
+        this.bookService = bookService;
+    }
+
+    @GetMapping("/customer")
+    public List getCustomers() {
+        return customerService.getCustomers();
+    }
+
+    @PostMapping("/book")
```

Comment:
If we change this class to only deal with books, we can change the `@RequestMapping` on the class to be `@RequestMapping("/api/book")`, and have `@PostMapping` (without any brackets) here, and `@GetMapping("/{id}")` below.

Reply:
Done